cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -15,6 +15,11 @@ spec:
         k8s-app: fluentd-gcp
         kubernetes.io/cluster-service: "true"
         version: v1.35
+  # This annotation ensures that fluentd does not get evicted if the node
```

Comment:
Please update also cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml. See comment from the first line.